cmd/idmd/db/db.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2021 The LibreGraph Authors.
+ */
+
+package db
```

Comment:
Hm, currently we've no plans for adding more databases. For now, to keep it simple, I'd rather rename the "db" command to "boltdb" instead of adding another layer.